tailwind.config.js

```diff
@@ -1,8 +1,24 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: ["./src/**/*.{html,ts}"],
+  content: ['./src/**/*.{html,ts}'],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        primary: '#222328',
+        secondary: '#969AB0',
+        accent: '#3545E9',
+        warning: '#FEFD54',
+        success: '#63C995',
+        error: '#E23D69',
+        bright: '#FFFFFF',
+        muted: '#7C9CBF',
+        shade: '#DBE2EA',
+      },
+    },
+    fontSize: {
+      h1: '72px',
```

Comment:
tailwind already has text-7xl = 72px. https://tailwindcss.com/docs/font-size
If you need to add 56px, let's use name that can match with the tailwind approach, to keep consistency around the project.
Because text-7xl and text-h3 are very different